app.py

```diff
@@ -1,43 +1,48 @@
 from flask import Flask, render_template, request, redirect, url_for
-import trello_api as trello
+import mongo_api as mongo
 from viewModels.indexViewModel import IndexViewModel
 from dotenv import find_dotenv, load_dotenv
+import os
 
 def create_app():
     app = Flask(__name__)
-    trello.load_all_list_ids()
+    mongoApi = mongo.MongoApi(
+        os.getenv('MONGO_USERNAME'),
+        os.getenv('MONGO_PASSWORD'),
+        os.getenv('MONGO_URL'),
+        os.getenv('DATABASE'))
 
     @app.route('/')
     def index():
-        items = trello.get_items()
+        items = mongoApi.get_items()
```

Comment:
Minor, but here's a good example of where we could prevent implementation details leaking - just making this variable name a little more generic (`item_storage`?) would save this class from having to update as many places in future, and we could essentially treat our storage solution as an interface that we only care about the specifics of on line 8, which you've demonstrated by keeping the same method names on MongoApi

Reply:
Thanks!